### PR TITLE
Add py36, V1.4.1 and rerender with 2.4.0

### PR DESCRIPTION
**Pull request by kmuehlbauer**

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@jakirkham Sorry, I totally lost [this one](https://github.com/conda-forge/shapelib-feedstock/pull/1). 

I tried to add py36 to the recipes build matrix and also bumped the version to 1.4.1. After that I rerendered with smithy 2.4.0. Hope this is all correct. If not, please let me know how to handle things.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,20 +16,20 @@ build:
   features:
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would add the following as well and re-render. The reason being Python 3.5 and 3.6 both use the same `vc` version. So we only need to build one of them. Easiest being Python 3.6 as we will likely drop Python 3.5 in the near future (when Python 3.7 comes out).

```yaml
  skip: true  # [win and py35]
```

**Comment by jakirkham** [comment]
No worries. Thanks for the ping.

Basically looks fine. Made a suggestion to slim the AppVeyor matrix a little bit. Otherwise LGTM. Please let me know if you have any more questions.

**Comment by kmuehlbauer** [comment]
Added per your suggestion 👍 Thanks for looking into this! Do I need to cancel the first builds, or is this done automatically? If this is OK from your side I'll merge this when the CI's are happy.

**Comment by kmuehlbauer** [comment]
Ahh, I see, appveyor canceled the build automatically...

**Comment by jakirkham** [comment]
So we added cancellation code for old PR builds on AppVeyor based on code from Julia's AppVeyor configuration a long time back. Around the beginning of this year, we rewrote that code in Python and generalized it to work on all CIs. It lives in `conda-forge-build-setup`. Trick is this code only runs when the build starts. At some point we should look at CI provided solutions to this problem. Some of these CI provided solutions popped up after adding that script. We just have to verify they only are used on PRs and not on feedstock branches.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Done above.

**Comment by kmuehlbauer** [comment]
Cool. I didn't follow that implementation. Nice solution to get the CI's workload down. CI-side cancellation methods would surely improve that.

**Comment by jakirkham** [comment]
Yep. Here's the [script]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/76607223907d2df8a44cc242418a0aa26f1fb6e7/recipe/ff_ci_pr_build.py ) if you are curious.

Yeah CI side implementations would be better still. At the time Travis CI had one in beta with an unpublished API. AppVeyor had one for a while, but we didn't use it for some reason that I have never known. Probably the same reason Julia didn't use AppVeyor's solution. CircleCI quietly added one afterwards, but their notions of PRs are pretty weak. So am not sure how well a CircleCI implementation would work or if it would just be more hassle. Would require someone to test these out to ensure they actually work for our use case and can be enabled via an API.

**Pull request merged**